src/null-safety/index.md

```diff
@@ -344,6 +344,12 @@ The Dart team is currently aware of the following issues:
   * Migration of the pub.dev packages owned by the Dart team
     is nearly complete, but a few are still missing. See pub.dev for
     [null-safe packages from the Dart team][ns-dart-pkgs].
+  * When publishing packages with a regular (non-dev) dependency on
```

Comment:
I wonder if we need to start with a sentence that explains that direct dependencies are not typical? Maybe something like:

Packages which have unit tests typically have a dev dependency on `package:test`. A few packages extend the framework in `package:test`, and may have a direct dependency. When publishing those packages...

Reply:
done